src/rickbox/app.py

```diff
@@ -1,12 +1,25 @@
 from flask import Flask
 
+from blueprints.register import register_all_blueprints
+from database import RecordNotFoundError
+from database.models.user import User
+
 app = Flask(__name__)
+register_all_blueprints(app)
 
 
 @app.route('/')
 def hello_world():
     return 'Hello, World!'
 
 
+@app.route('/get_user_by_id')
+def get_user_by_id():
+    try:
+        return User().columns
+    except RecordNotFoundError as err:
+        return str(err)
+
+
```

Comment:
This is intentionally bad so that it doesn't sneak in.

How would you like endpoint code to be organized? Are we going MVC? MVVM?

Reply:
I'm not sure if either is completely appropriate. Maybe a hybrid of them both? `app.py` should act as our controller, and the models as models. Since we're only doing an API for now we won't have a view. When we create the SPA then I suppose we'll have a view then. For now my question is where to put the  stored procedure functions. Should they be static functions in their respective models, in a dedicated database helper file, or something else?